lib/golomb/golomb.js

```diff
@@ -234,7 +234,6 @@ class Golomb {
    */
 
   toRaw() {
-    assert(this.p === 19);
```

Comment:
I think it will be a good idea to check if p < 32, to avoid integer overflow.

Reply:
Where do you suggest the check be placed? I think all functions constructing the filter in golomb.js have the relevant assertion checks(for e.g the `fromItems` function).
 I believe placing a check for p < 32 in the `toRaw` would be redundant because by the time we call toRaw the filter would already have been constructed after passing the relevant assertion checks.

Reply:
it would be a good idea to add the check just in case the value of p gets modified before serialization.

Reply:
If the value of p gets modified before serialization then the filter is corrupted anyway(thereby not of any use) regardless if p is less than 32 or not.

Reply:
I see, makes sense.